tests/integration/cli/classification/test_classification.py

```diff
@@ -519,3 +521,58 @@ def test_pot_optimize(self, template, tmp_dir_path):
     @pytest.mark.parametrize("template", templates, ids=templates_ids)
     def test_pot_eval(self, template, tmp_dir_path):
         pot_eval_testing(template, tmp_dir_path, otx_dir, args_h)
+
+
+# tmp: create & remove data.yaml to only use train-data-roots
```

Comment:
If this temporary function is needed to test `self-sl` task, how about making separate `test_self.py`? not included in `test_classification.py`.

Reply:
For the semi-supervised learning, we agreed on one test file for one task, however if self-sl has many difference with other tasks, then it's okay to make separate test file

Reply:
I think it is unnecessary to create separate test file because of this temp function.
It will be removed when single argument (e.g. --train-data-roots) can be used.

Reply:
I prefer that temporary variables or functions should not be included in the existing one even though they will be deleted.